__tests__/api.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest')
+
+const app = require('../app')
+const db = require('../db/connection')
+const testData = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed') 
+
+afterAll(() => db.end)
+beforeEach(() => seed(testData))
+
+describe('API testing', () => {
+    describe('GET Requests', () => {
+        test('200 - Should return correct values within the topics API. Format should be an array of objects for slug and description properties', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body: { topics } }) => {
+                expect(topics).toBeInstanceOf(Array);
+                expect(topics).toHaveLength(3);
```

Comment:
good job checking the length of the array here

Reply:
Thank you!